appinventor/docs/reference/components/userinterface.html

```diff
@@ -667,7 +667,10 @@ <h3>Properties</h3>
                  </dl>
 
                  <h3>Events</h3>
-                 none
+                 <dl>
+		   <dt><code>Click()</code></dt>
+		   <dd>User tapped and released an image</dd>
```

Comment:
You may want to make this text match the description given in the `@SimpleEvent` annotation in the sources.